doc/releases/0.98.0.rst

```diff
@@ -0,0 +1,133 @@
+    * High-pass spatial filter
+
+  * | Add subfolder for sorter output ("output_folder/sorter_output") and :code:`sorting_info` field to Sorting,
+    | so that "sorter_output" can be deleted without deleting metadata and :code:`sorting_info` (#1149, #1150)
+
+  * Refactor Quality Metrics module (#1155, #1177, #1197):
+
+    * Improve parameter handling
+    * Unified naming and conventions
+    * Add missing metrics: drift, amp medians, sliding_rp_violations
+
+  * Extended Motion estimation and motion correction, which is now ready to use (#1129, #1108, #1183)
+
+  * Removed deprecated functions/modules (#1261):
+
+    * :code:`toolkit` module
+    * :code:`symmetrize` argument from compute_correlograms
+    * :code:`localize_units` from postprocessing
+
+  * Major rewrite of the docs, with extensive module-specific documentation (#1264, #1265, #1267, #1268, #1269, #1273, #1275, #1276, #1277, #1281, #1284, #1285, #1287, #1288, #1291)
+
+core:
+  * Added possibility of extracting unfiltered waveforms (#1046)
+  * Prefer :code:`folder`` argument and make :code:`zarr_path`` deprecated in :code:`save()` function (#1049)
+  * Fix bug 'int16' is not a JSON serializable (#1069)
+  * Add waveform extractor :code:`save()` function and Zarr backend (#1079)
+  * :code:`ChunkRecordingExecutor`: add max_thread_per_process to limit the number of thread using the module threadpool_limits (#1086)
+  * Correct channel selection in tests when mono-channel recording (#1096)
+  * Add :code:`split_job_kwargs()` function (#1110)
+  * Improve handling of not dumpable objects (#1131)
+  * Implement ChannelSparsity object in core (#1136)
+  * Handle sparsity inside WaveformExtractor (#1152)
+  * Use absolute folder in NpySnippetsFolder (#1171)
+  * Add global_job_kwargs functions (#1176)
+  * Propagate probe annotations to recording (#1208)
+  * Serialization for recording and sorting objects (#1227)
+  * Extend WaveformExtractor rec_attributes and propagate recordingless mode (#1258)
+  * Improve serialization (#1227, #1272)
+
+extractors:
+  * NWBRecordingExtractor offset extraction (#1057)
+  * fsspec for streaming NWB data (#1090)
+  * Optimizing Phy-based extractors (#1153)
+  * Add CellExplorerSortingExtractor (#1158)
+  * Fix neuralynx sorting function and NeoBaseSorting when no streams (#1171)
+  * Add option to load synchronized timestamps in OpenEphys (#1198)
+  * Streaming IBL recordings (#1204, #1251)
+  * Load phy properties from cluster_info.csv (#1210)
+  * Fix the sample_shifts for spikeglx when not all channel are saved (#1232)
+
+preprocessing:
+  * Add IBL channel interpolation to preprocessing (#1025)
+  * Add IBL high pass spatial filter to preprocessing (#1059)
+  * Add IBL bad channel detection (#1070)
+  * Add get_spatial_interpolation_kernel in preprocessing (#1074)
+  * Add median/average artefact modes to :code:`remove_artifacts` (#1106, #1143, #1250)
+  * Fix phase-shift dtype (#1172)
+  * Add correct_lsb function (#1262)
+
```

Comment:
Maybe the padding and the dtype here?

Reply:
I forgot to save the file! It was supposed to be a blank file eheheh

Reply:
I'll start with the proper release notes tomorrow